src/tests/routes/index.js

```diff
@@ -1,3 +1,6 @@
 import './comments.test';
 import './404.test';
 import './auth.test';
+import './commentlike.test';
```

Comment:
It's better to rename this file to `commentLike.test.js`., Camel Case as other files

Reply:
sure.. Thank u.. Let me check